var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -43,10 +43,6 @@ class Llvm(CMakePackage):
     version('3.0', 'a8e5f5f1c1adebae7b4a654c376a6005',
             url='http://llvm.org/releases/3.0/llvm-3.0.tar.gz')
 
-    variant('debug', default=False,
-            description="Build a debug version of LLVM, this increases "
```

Comment:
maybe keep this comment somewhere in this package so that is someone has issues, he could look through the code.